packages/frontend/src/view/pages/state-update/common.ts

```diff
@@ -3,11 +3,13 @@ export const getBalanceChangeTableProps = (id: string) => ({
   entryShortNamePlural: 'changes',
   entryLongNamePlural: 'balance changes',
   link: `/state-updates/${id}/balance-changes`,
+  description: `Shows table of balance changes for #${id} state update`,
 })
 
 export const getTransactionTableProps = (id: string) => ({
   title: 'Included forced transactions',
   entryShortNamePlural: 'transactions',
   entryLongNamePlural: 'forced transactions',
   link: `/state-updates/${id}/transactions`,
+  description: `Shows table of transactions included in #${id} state update`,
```

Comment:
Check next comments. Don't use "Shows..." text and "table". Just "Transactions included in .."